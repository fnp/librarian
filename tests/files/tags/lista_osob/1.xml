<autor_utworu>Sofokles</autor_utworu>
<nazwa_utworu>Antygona</nazwa_utworu>

<lista_osob>

<naglowek_listy>OSOBY DRAMATU</naglowek_listy>

<lista_osoba>ANTYGONA, córka Edypa</lista_osoba>
<lista_osoba>ISMENA, jej siostra</lista_osoba>
<lista_osoba>CHÓR TEBANSKICH STARCÓW</lista_osoba>
<lista_osoba>KREON, król Teb</lista_osoba>
<lista_osoba>STRAŻNIK</lista_osoba>
<lista_osoba>HAJMON, syn Kreona</lista_osoba>
<lista_osoba>TYREZJASZ, wróżbita</lista_osoba>
<lista_osoba>POSŁANIEC.</lista_osoba>
<lista_osoba>EURYDYKA, żona Kreona</lista_osoba>
<lista_osoba>POSŁANIEC drugi</lista_osoba>

</lista_osob>

<miejsce_czas>Rzecz dzieje się przed dworcem królewskim w Tebach</miejsce_czas>
