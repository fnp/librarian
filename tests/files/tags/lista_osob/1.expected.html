<div id="book-text">
<h1>
<span class="wl author" id="i1">Sofokles</span><span class="wl title" id="i2">Antygona</span>
</h1>
<div class="person-list" id="i14">
<h3 class="wl" id="i3">OSOBY DRAMATU</h3>
<ol>
<li class="wl" id="i4">ANTYGONA, córka Edypa</li>
<li class="wl" id="i5">ISMENA, jej siostra</li>
<li class="wl" id="i6">CHÓR TEBANSKICH STARCÓW</li>
<li class="wl" id="i7">KREON, król Teb</li>
<li class="wl" id="i8">STRAŻNIK</li>
<li class="wl" id="i9">HAJMON, syn Kreona</li>
<li class="wl" id="i10">TYREZJASZ, wróżbita</li>
<li class="wl" id="i11">POSŁANIEC.</li>
<li class="wl" id="i12">EURYDYKA, żona Kreona</li>
<li class="wl" id="i13">POSŁANIEC drugi</li>
</ol>
</div>
<p class="wl place-and-time" id="i15">Rzecz dzieje się przed dworcem królewskim w Tebach</p>
</div>
